core/src/main/resources/jenkins/model/Jenkins/configure.jelly

```diff
@@ -31,15 +31,20 @@ THE SOFTWARE.
   <st:include page="sidepanel.jelly" />
   <f:breadcrumb-config-outline />
   <l:main-panel>
+    <div class="jenkins-app-bar">
+      <div class="jenkins-app-bar__content">
```

Comment:
This should just be a custom tag. There's no reason for blocks like this to be copied & pasted all over.

Reply:
https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/layout/app-bar.jelly ?

Reply:
You're right :) I've updated this, thanks!